src/readpst.c

```diff
@@ -418,8 +425,10 @@ void process(pst_item *outeritem, pst_desc_tree *d_ptr)
         }
         pst_freeItem(item);
     }
+    success &= ff.found_count == ff.stored_count;
```

Comment:
A client reported a PST that had emails that were not getting extracted - they were simply not found in this traversal (although they were extracted by Nuix).  The client ran the PST through a "fixer" (perhaps SCANPST) and that resolved the issue.  These counts were correct in the original pst, so I think marking the doc as an error is a good compromise (assuming they are generally reliable/correct).

This will also raise an error if there are skipped/unknown items, which is probably a good idea, although we'll probably want to see how this turns out in practice (also I can test across some sample psts we've see in the past and look for errors).